policies.tex

```diff
@@ -73,7 +73,8 @@ \subsection{}Club meetings may be called by the EC, or by petition of five votin
 \section{}The class shall be open to anyone, except that the EC may, at its discretion and for up to one class per year, limit the class to MIT students only.
 \section{}The Class Coordinator, after notifying the EC, may remove people from the class if they are interfering with the class or preventing other class members from learning.
 \section{}The EC must approve the list of class graduates. The Class Coordinator shall provide the EC with a list of class members, and provide a recommendation of which should graduate, which should not, and which deserve discussion.
-\section{}A class member must dance club level or be expected to dance club level soon to graduate. 
+\section{}A class member must dance club level or be expected to dance club level soon to graduate.
+\section{}Running the class often requires missing the first and second club tips (due to walkthroughs and class meeting, respectively). In recognition of this, the class coordinator and up to three class assistants they designate shall receive a free subscription for the terms the class is in session. The class coordinator must inform the EC who should receive free subscriptions. Each person receiving a free subscription must be missing at least half of club tips due to their service on the class committee.
```

Comment:
s/a free subscription/free subscriptions/
That sentence also seems like it should somehow be combined with the following one.